data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
             updatedBy: "Alexander", updatedAt: "2020-02-03",
         },
     },
+    {
+        meta: {
+            name: "12/06/2020 Olympia",
+            link: "https://www.startnext.com/12062020",
+            keywords: ["petition", "networking", "event", "grassroots"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "One-time event where up to 90,000+ people come together to rent the Olympia stadium Berlin to sign various petitions."
+            + " In Germany, petitions must be discussed by parliament if they get more than 50,000 signatures. In addition there will be talks and presentations on currently pressing social and environmental problems.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "German Zero",
+            link: "https://germanzero.de/",
+            keywords: ["policy", "legislation"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "GermanZero is creating a climate action plan to meet Germany's 1.5C commitment that will be turned into proposed legislation before 2022. A nation-wide campaign will then try to get that legislation passed in parliament.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Hamburg muss handeln",
```

Comment:
These is also 'München muss handeln' https://muenchen-muss-handeln.de/ 
and 'Augsburg handelt' https://augsburg-handelt.de/
All with the same idea. They also should be represented.

Reply:
Agree. Unless you'd merge them into one (which would then make it impossible to tag their locations) I'd consider them additional features and not changes about this PR. I added them to [Content to Add > Initiatives](https://docs.google.com/spreadsheets/d/1mN4i9pIJuqvhFS1s-EB4l4ICs7gmW8gZzQyqH8jJD8Y/edit#gid=267342746).

Reply:
Agree.